modules/installation-custom-alibaba-vpc.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_alibaba/installing-alibaba-vpc.adoc
+
+:_content-type: CONCEPT
+[id="installation-custom-alibaba-vpc_{context}"]
+= About using a custom VPC
+
+In {product-title} {product-version}, you can deploy a cluster into existing subnets in an existing Virtual Private Cloud (VPC) in the Alibaba Cloud Platform. By deploying {product-title} into an existing Alibaba VPC, you might be able to avoid limit constraints in new accounts or more easily abide by the operational constraints that adhere to your company's guidelines set. If you cannot obtain the infrastructure creation permissions that are required to create the VPC yourself, use this installation option. You must configure networking using vSwitches.
+
+[id="installation-custom-alibaba-vpc-requirements_{context}"]
+== Requirements for using your VPC
+
+The union of the VPC CIDR block and the machine network CIDR must be non-empty. The vSwitches must be within the machine network.
+
+The installation program does not create the following components:
+
+* VPC
+* vSwitches
+* Route table
+* NAT gateway
+
+include::snippets/custom-dns-server.adoc[]
+
+[id="installation-custom-alibaba-vpc-validation_{context}"]
+== VPC validation
+
+To ensure that the vSwitches you provide are suitable, the installation program confirms the following data:
+
+* All the vSwitches that you specify must exist.
+* You have provided one or more vSwitches for control plane machines and compute machines.
+* The vSwitch's CIDRs belong to the machine CIDR that you specified.
```

Comment:
```suggestion
* The vSwitches' CIDRs belong to the machine CIDR that you specified.
```
Nit in case there are multiple vSwitches

Reply:
Change made, although...I need to check the style guide on that trailing apostrophe. I'm not sure we need that?

Reply:
I'm not sure either.. I was thinking of it in plural possessive, "The CIDRs of the vSwitches" aka "The vSwitches' CIDRs" ... may be best to just reword as "The CIDR of each vSwitch belongs to..." or some such. Up to you